lib/graphics.cc

```diff
@@ -27,4 +28,70 @@ int DrawText(Canvas *c, const Font &font,
   }
   return x - start_x;
 }
+    
+    
+void DrawCircle(Canvas *c, int32_t x0, int32_t y0, int32_t radius, uint8_t red, uint8_t green, uint8_t blue){
```

Comment:
At the end of this method header, add a space before the curly brace
  `...blue) {`